src/components/tasks/card/card.module.scss

```diff
@@ -97,7 +100,46 @@ $black: #252525;
 .progressYellow {
   background-color: $yellow;
 }
+
 .editButton {
   text-align: left;
   cursor: pointer;
 }
+
+.dropdownMenu {
+  position: absolute;
+  right: 0;
+  top: 90%;
+  z-index: 10;
+}
+
+.inputAssigneeSearch {
+  margin-bottom: .5rem;
+}
+
+.assigneeMenu {
+  border-radius: .25rem;
+  background-color: #ffffff;
+  padding: .5rem;
+  box-shadow: 0 0 20px $black;
+
+  .assigneeList {
+    margin: 0;
+    padding: 0;
+  }
+
+  .userName {
+    margin: 0;
```

Comment:
Is it necessary here to set `margin` explicitly ?

Reply:
removed not necessary